### PR TITLE
added wordpress plugin

### LINKED ISSUE #5138: [CAL-213] Official Cal.com Wordpress Plugin
> From [Linear-GitHub Sync](https://synclinear.com)

### PR DESCRIPTION
**Pull request by PeerRich**
- [x] add all 3 embed options
- [x] create wordpress plugin page
- [x] link to plugin page
- [x] create and add screenshots

CleanShot 2022-10-21 at 21 40 09@2x

### CODE REVIEW

**Comment by zomars** [comment]
ryan reynolds hd GIF

**Comment by PeerRich** [comment]
closing for now -- not a priority

File: packages/app-store/wordpress/extensions/EventTypeAppCard.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+
+import { useAppContextWithSchema } from "@calcom/app-store/EventTypeAppContext";
+import AppCard from "@calcom/app-store/_components/AppCard";
+import type { EventTypeAppCardComponent } from "@calcom/app-store/types";
+import { Icon } from "@calcom/ui";
+
+import { appDataSchema } from "../zod";
+
+const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({ eventType, app }) {
```
Comment (JeroenReumkens) [inline packages/app-store/wordpress/extensions/EventTypeAppCard.tsx:10]: Is this file required?

File: packages/app-store/wordpress/zod.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
```
Comment (JeroenReumkens) [inline packages/app-store/wordpress/zod.ts:1]: Is this file required?

**Review by JeroenReumkens — APPROVED** [review]
Two files seem to be part of the template but not really necessary? 
Apart from that all fine to me :v:

File: packages/app-store/wordpress/extensions/EventTypeAppCard.tsx (hunk 1 above)
Comment (PeerRich) [reply packages/app-store/wordpress/extensions/EventTypeAppCard.tsx:10]: ah right, it's not

Comment (PeerRich) [reply packages/app-store/wordpress/extensions/EventTypeAppCard.tsx:10]: removed
